### ISSUE TITLE
Missing setup information

### ISSUE THREAD

**Issue by clonejo** [Kasalehlia/dynmenu#3]
The README lacks information about how to set up dynmenu.

My approach so far (using code from pull-request #2):

``` bash
git clone <repo url> # to ~/.menu
cat menu.sql | sqlite3 main.sqlite
{ echo "BEGIN;"; dmenu_path|{while read c; do echo "INSERT INTO commands (cmd) VALUES ('$c');" ; done}; echo "END;" } | sqlite3 test.sqlite
sqlite3 main.sqlite
# change dmenu hotkey to ~/.menu/menu main
```

**Comment by clonejo** [comment]
updated the code for filling the database, much faster now

**Comment by clonejo** [comment]
Fixed with PR #4.
